docs/eigenda/operator-guides/upgrades/system-upgrades.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 2
+---
+
```

Comment:
This part is the more challenging part for operators. I'll try to flesh out some thing and add here later as well.

Reply:
That would be great